src/_data/registerdemos.js

```diff
@@ -13,4 +13,18 @@ module.exports = [
     lang: 'fr',
     link: 'https://events.teams.microsoft.com/event/a9bfe87d-4692-465c-a6db-a9c5ef377175@9ed55846-8a81-4246-acd8-b1a01abfc0d1'
   }
-]
\ No newline at end of file
+{
+    date: "2025-02-18",
+    starttime: "13:00",
+    endtime: "14:00",
+    lang: 'en',
+    link: 'https://events.teams.microsoft.com/event/ca8b7c9d-8fb9-4eaf-8240-c7127b7650a7@9ed55846-8a81-4246-acd8-b1a01abfc0d1'
+  }
+{
```

Comment:
Make sure to add a comma between each event object. You do not need one after the last event object in the array

```suggestion
  },
  {
```